extensions/interactions/MultipleChoiceInput/MultipleChoiceInput.py

```diff
@@ -66,6 +66,15 @@ class MultipleChoiceInput(base.BaseInteraction):
         'default_value': [{
             'content_id': None,
             'html': ''
+        }, {
+            'content_id': None,
+            'html': ''
+        }, {
+            'content_id': None,
+            'html': ''
+        }, {
+            'content_id': None,
+            'html': ''
         }],
```

Comment:
Do you think it's better to have the default value be set to 4 items? Or to have it set to 1 item, which immediately shows the message that you need at least 4 items?

Reply:
I think we should pass these 4 items as a default value which will make it obvious for the user to enter 4 values and in case if he cancels one of them then the error message will appear. This way it will be more interactive but please do let me know your thoughts on this.
Thanks @EricZLou

Reply:
I believe the RTE component doesn't let you delete items if you have 4 items right? So you'll never be able to go below 4 so the error message would never show up?

Reply:
Sorry for the late reply.
Yeah, you were right, I have updated the validation, and this way it's more clear.
Thanks
Here is the SS
![Screenshot from 2021-10-02 23-01-27](https://user-images.githubusercontent.com/44300735/135726859-72c00f47-4eaa-499a-9ee4-4de98bec261d.png)

Reply:
@EricZLou, I have one thing that I want to ask, so I have to update the test files too because of the `min_value` right? but now if we'll pass only one default value then how it's gonna work?

Reply:
I think it is fine the way you have it now! Just need to fix some failing frontend tests. And we'll get input from the other reviewers on whether this is a good plan: setting min value to 1 even though the min value is really 4, just so it's easier for the user to understand.

Reply:
I think we can do that because we are doing the same in the `DragAndDrop` interaction also I want to ask that I can see tests that have `MultipleChoice` and other interactions as input but I didn't see any `DragAndDrop` input interaction as a test, can you point me to some files that have so that I can compare the failing frontend tests.
Thanks @EricZLou